### ISSUE TITLE
org.kde.KStyle.Adwaita 5.15-22.08 not loading Adwaita-dark for application

### ISSUE THREAD

**Issue by Eonfge** [flathub/org.kde.KStyle.Adwaita#12]
As a follow-up to #11, I must inform that the theme is not loading properly for Qbittorrent.

### Qbittorrent
```
qBittorrent - An open-source Bittorrent client

          ID: org.qbittorrent.qBittorrent
         Ref: app/org.qbittorrent.qBittorrent/x86_64/stable
        Arch: x86_64
      Branch: stable
     Version: 4.4.5
     License: GPL-2.0 and OpenSSL
      Origin: flathub
  Collection: org.flathub.Stable
Installation: system
   Installed: 14,7 MB
     Runtime: org.kde.Platform/x86_64/5.15-22.08
         Sdk: org.kde.Sdk/x86_64/5.15-22.08

      Commit: 997c137fc8fa75c2ba2d2b0e6a85ea3e9aa4ae28940f8b7dd755cec8013a92aa
      Parent: caaa3fc438f7808badb445cf0dcbcc77aef574ff5757e07d0544e30bd56ba313
     Subject: Bump SDK version to 5.15-22.08 (#90) (2f094d86)
        Date: 2022-09-10 04:50:55 +0000
```

### Adwaita
```

Adwaita theme - Adwaita widget theme matching the GNOME adwaita theme

          ID: org.kde.KStyle.Adwaita
         Ref: runtime/org.kde.KStyle.Adwaita/x86_64/5.15-22.08
        Arch: x86_64
      Branch: 5.15-22.08
      Origin: flathub
  Collection: org.flathub.Stable
Installation: system
   Installed: 16,2 MB

      Commit: 7496a6a69ee3a690727994acb2f37840b5aef1c9551803e0e6135c8e727528f4
     Subject: Reformat (a77e9b78)
        Date: 2022-09-12 08:28:53 +0000
```

When using another Qt application like kid3, it does work. For reference, my host system is Fedora 36 Workstation, with dark-theme enabled.

![Screenshot from 2022-09-12 17-08-40](https://user-images.githubusercontent.com/33983090/189689908-01602527-1493-496c-8a4e-3b823b397503.png)

## Testing
I looked at the documentation of [Adwaita-qt](https://github.com/FedoraQt/adwaita-qt) and I found that I have to manually start the application with `-style Adwaita-dark` for it to take effect. Not sure what the appropriate cause of action would be here, but I do hope that this can be fixed. Should I report it upstream?

**Comment by travier** [comment]
Was this working before or is this a new issue?

**Comment by Eonfge** [comment]
Yes. I can roll back to the pre-22.08 version with this command and then I get the dark-mode back

`sudo flatpak update --commit=caaa3fc438f7808badb445cf0dcbcc77aef574ff5757e07d0544e30bd56ba313 org.qbittorrent.qBittorrent`

**Comment by q234rty** [comment]
Could this be the same issue as https://github.com/flathub/org.kde.PlatformTheme.QGnomePlatform/issues/7 ?

**Comment by q234rty** [comment]
i.e. @Eonfge do you have [org.kde.WaylandDecoration.QGnomePlatform-decoration](https://github.com/flathub/org.kde.WaylandDecoration.QGnomePlatform-decoration) or [org.kde.PlatformTheme.QGnomePlatform](https://github.com/flathub/org.kde.PlatformTheme.QGnomePlatform) installed?

**Comment by Eonfge** [comment]
Quite a few;

```
QGnomePlatform	org.kde.PlatformTheme.QGnomePlatform		5.15-21.08	flathub	system
QGnomePlatform	org.kde.PlatformTheme.QGnomePlatform		6.2	flathub	system
QGnomePlatform	org.kde.PlatformTheme.QGnomePlatform		6.3	flathub	system
QGnomePlatform-decoration	org.kde.WaylandDecoration.QGnomePlatform-decoration		5.15-21.08	flathub	system
QGnomePlatform-decoration	org.kde.WaylandDecoration.QGnomePlatform-decoration		6.2	flathub	system
QGnomePlatform-decoration	org.kde.WaylandDecoration.QGnomePlatform-decoration		6.3	flathub	system
```

**Comment by q234rty** [comment]
Could you try installing the 5.15-22.08 branch for both of these and downgrading qbittorrent back to commit `997c137fc8fa75c2ba2d2b0e6a85ea3e9aa4ae28940f8b7dd755cec8013a92aa` which uses the 5.15-22.08 runtime, to see if things work?

**Comment by Eonfge** [comment]
Was already working on it!

I installed 5.15-22.08 and switched  QBittorrent to `997c137fc8fa75c2ba2d2b0e6a85ea3e9aa4ae28940f8b7dd755cec8013a92aa`, and I now have dark themes! 

Perhaps something wrong with the dependencies for KStyle Adwaita?

**Comment by Eonfge** [comment]
In other news, I found a second application [Proton Up](https://flathub.org/apps/details/net.davidotek.pupgui2) that uses KDE 6.3 and that application also doesn't adhere to KStyle Adwaita. Is that related or different?

**Comment by q234rty** [comment]
Kstyle adwaita is just a qt theme: it's not loaded by default and you have to use env vars like `QT_STYLE_OVERRIDE=Adwaita-dark` or arguments `-style Adwaita-dark`. However qgnomeplatform is loaded by default under GNOME and doesn't need kstyle Adwaita to work(the style is bundled).

**Comment by Erick555** [comment]
is this fixed?
